### PR TITLE
Added instance methods for convenience

### PR DESCRIPTION
**Pull request by rickycpadilla**

### CODE REVIEW

**Review by blaynebayer — APPROVED** [review]
looks good!

File: Sources/Passage/PassageAuth.swift [hunk 1]
```diff
@@ -190,6 +190,34 @@ public class PassageAuth {
         return authResult
     }
     
+    /// Activates a One-Time Passcode
+    ///
+    /// Upon successful activation the tokens will be stored in the tokenStore on the instance.
+    ///
+    /// - Parameter otp: string - The OTP provided by your user
+    /// - Parameter otpId: string - The OTP id returned from login or register method
+    /// - Returns: ``AuthResult`` The AuthResult object contains an authentication token (JWT) and redirect URL. The auth token should be used on all subsequent authenticated requests to the app. The redirect URL specifies the route that users should be redirected to after completed registration or login
```
Comment (macleanevans) [inline Sources/Passage/PassageAuth.swift:199]: Is there any way to make this reusable? Seems to be used a bunch of places and would it nice if we could just store it as a string value - I don't know how you'd do that in this syntax though.

File: Sources/Passage/PassageAuth.swift [hunk 2]
```diff
@@ -190,6 +190,34 @@ public class PassageAuth {
...
+    /// - Throws: ``PassageAPIError``, ``PassageError``
+    public func oneTimePasscodeActivate(otp: String, otpId: String) async throws -> AuthResult {
+        clearTokens()
```
[the remark this answers is not in the document]
Comment (rickycpadilla) [reply Sources/Passage/PassageAuth.swift:202]: This matches the pattern of the other auth instance method calls - verifies the tokens are cleared before authenticating the user who's trying to login (which could be a different user theoretically)

File: Sources/Passage/PassageAuth.swift (hunk 1 above)
Comment (rickycpadilla) [reply Sources/Passage/PassageAuth.swift:199]: Not that I'm aware of. But I am hoping to spend some time cleaning up inline docs when we can budget time for iOS work 🤞

**Review by macleanevans — APPROVED** [review]

**Pull request merged**